app/src/test/kotlin/com/mapzen/erasermap/model/TestAppSettings.kt

```diff
@@ -2,8 +2,10 @@ package com.mapzen.erasermap.model
 
 import android.content.Context
 import android.location.Location
+import com.mapzen.android.MapzenMap
 import com.mapzen.erasermap.dummy.TestHelper
 import com.mapzen.pelias.SavedSearch
+import com.mapzen.tangram.MapController
```

Comment:
Unused import? I sure wish Kotlin had a version of Checkstyle to validate this stuff for us :/